Makefile

```diff
@@ -1,10 +1,8 @@
 export ARCHS = arm64 arm64e
-export TARGET = iphone:clang:14.4:13.0
-export SYSROOT = $(THEOS)/sdks/iPhoneOS14.4.sdk
-export PREFIX = $(THEOS)/toolchain/Xcode.xctoolchain/usr/bin/
+export TARGET = iphone:clang:14.5:13.0
```

Comment:
I'm not going to merge Makefile changes that are specific to your setup.

Reply:
Of course.